### PR TITLE
refactor: use title tag in pages instead of _document

### PR DESCRIPTION
**Pull request by iShibi**
### 👨‍💻 Changes proposed

Next.js warns not to use the title tag in the `_document` page instead each page should have its own title tag. The new title tags are inspired by GitHub's document titles.

### CODE REVIEW

**Comment by subhoghoshX** [comment]
Thank you. I think @ykdojo prefers `defaang` over `Defaang`.

**Comment by iShibi** [comment]
> Thank you. I think @ykdojo prefers `defaang` over `Defaang`.

True. I did try lowercase however, it didn't look good. `defaang` in lowercase looks nice as logo but when writing it as a text it just doesn't provide that feel. An example of this exact situation is Stack Overflow, they use lowercase in their logo but capitalize their name when it is used in a text. Some other examples are `AWS` and `Netlify`.

I'm not an expert here but a capitalized name does give some weight to the Word and makes it stand out in a text.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ const Home: NextPage = () => {
 	return (
 		<div>
 			<Head>
-				<title>defaang</title>
+				<title>Defaang: Insert tagline here</title>
```
Comment (ykdojo) [inline src/pages/index.tsx:9]: defaang: recently-asked interview questions at FAANG+, collected & curated

**Review by ykdojo — CHANGES_REQUESTED** [review]

**Comment by ykdojo** [comment]
@iShibi I see what you're saying, but I think lowercasing everything is ok - at least for now.

It's untraditional, but it appears more friendly to me, at least.

There's already [a discussion](https://github.com/ykdojo/defaang/discussions/53) about it, but feel free to start a new one, too.

**Comment by iShibi** [comment]
@ykdojo lowercase it is, then.

**Review by ykdojo — APPROVED** [review]
looks good, thank you

**Pull request merged**